src/grg_sphinx_theme/theme/grg_sphinx_theme/team.html

```diff
@@ -1,4 +1,5 @@
 {% extends "grg_sphinx_theme/layout.html" %}
+{% set title = "Team" %}
```

Comment:
can you change it to "DIPY TEAM"

Reply:
It will create an issue for the people who wants to reuse it.
Let me create this as a variable to set whatever the theme user wants to use

@skoudoro